server/controllers/plugins.js

```diff
@@ -0,0 +1,54 @@
+const pluginList = require('../plugin')
```

Comment:
Hey Gideon,
I think controllers are from a different issue, #53

Reply:
thanks @tejasj1718 this has been corrected